src/app/components/Embeds/EmbedHtml/index.stories.tsx

```diff
@@ -21,6 +21,9 @@ export const IDT2QuotePic = () => (
   <EmbedHtml embeddableContent={quotePicEmbedHtml} />
 );
 
-export const Riddle = () => <EmbedHtml embeddableContent={riddleHTML} />;
+export const Riddle = {
+  render: () => <EmbedHtml embeddableContent={riddleHTML} />,
```

Comment:
Oh nice! Glad we managed to fix this 👍

Reply:
I think we still have these disabled on Chromatic (line 13). Is the intention to renable them?